tests/test_other.py

```diff
@@ -7256,6 +7256,13 @@ def test_side_module_transitive_deps(self):
     err = self.expect_fail(final_link)
     self.assertContained('error: libside2.wasm: shared library dependency not found: `libside1.wasm`', err)
 
+  def test_side_module_folder_deps(self):
+    # Build side modules in a subfolder
+    os.mkdir('subdir')
+    self.run_process([EMCC, test_file('hello_world.c'), '-s', 'SIDE_MODULE', '-o', 'subdir/libside1.so'])
+    self.run_process([EMCC, test_file('hello_world.c'), '-s', 'SIDE_MODULE', '-o', 'subdir/libside2.so', '-L', 'subdir', '-lside1'])
+    self.run_process([EMCC, test_file('hello_world.c'), '-s', 'MAIN_MODULE', '-o', 'subdir/main.js', '-L', 'subdir', '-lside2'])
```

Comment:
Nice!   

Maybe `-o main.js` to that the final output is not itself in the subdir, just to make sure that works too?

Reply:
Sounds good, changed to -o main.js